scripts/openplanner/import-speakers-sessions-schedule.ts

```diff
@@ -137,55 +137,38 @@ export const importTeam = async (team: TeamMember[]) => {
 async function deleteCollection(collectionPath: string, batchSize: number = 100) {
   const collectionRef = firestore.collection(collectionPath)
   const query = collectionRef.orderBy('__name__').limit(batchSize)
-
-  return new Promise((resolve, reject) => {
-    deleteQueryBatch(query, resolve).catch(reject)
-  })
-}
-
-async function deleteQueryBatch(query: firestoreDep.Query, resolve: (value?: any) => any) {
   const snapshot = await query.get()
-
-  const batchSize = snapshot.size
-  if (batchSize === 0) {
-    // When there are no documents left, we are done
-    resolve()
-    return
+  for (const doc of snapshot.docs) {
+    await firestore.recursiveDelete(doc.ref);
```

Comment:
oh, this doesn't use batch :(

Reply:
Yes I spent a lot of time trying to do a batch transaction but in the end nothing was working.... 
From what I understood, this method uses a bulkWriter under the hood

Reply:
OKI! ❤️